### ISSUE TITLE
v3: String with escape character causes error

### ISSUE THREAD

**Issue by mikefarah** [go-yaml/yaml#871]
Given a valid yaml like:
```yaml
description: "Grants access to read\/write ip prefix settings"
```

This causes an error (due to `\/`): error: yaml: found unknown escape character

See playground here:

https://go.dev/play/p/Y8Kf_G_UrAI

**Comment by perlpunk** [comment]
There are already issues and PRs open about that:
* https://github.com/go-yaml/yaml/issues/840
* https://github.com/go-yaml/yaml/issues/797
* https://github.com/go-yaml/yaml/pull/862
* https://github.com/go-yaml/yaml/pull/841

**Comment by mikefarah** [comment]
Oops cheers 👍🏼

**Comment by michacassola** [comment]
> There are already issues and PRs open about that:
> 
> * [Escaped forward slashes are not recognized #840](https://github.com/go-yaml/yaml/issues/840)
> * [Error on optional string escapes, even though valid in JSON #797](https://github.com/go-yaml/yaml/issues/797)
> * [fix: Support escaped slash in strings #862](https://github.com/go-yaml/yaml/pull/862)
> * [fix: recognize escaped forward slash #841](https://github.com/go-yaml/yaml/pull/841)

Cool!
Any ETA for a fix?
